aten/src/ATen/core/dispatch/OperatorEntry.h

```diff
@@ -217,8 +217,8 @@ class TORCH_API OperatorEntry final {
   // currently not high-pri.
   ska::flat_hash_map<DispatchKey, std::list<AnnotatedKernel>> kernels_;
 
-  AnnotatedKernel missingKernel_;
```

Comment:
oh this wasn't static, so I guess we used to have 1000+  default-initialized AnnotatedKernel objects lying around? Maybe this'll improve binary size a bit